### DISCUSSION TITLE
"total count" is wrong on paginated pages

### DISCUSSION THREAD

**Discussion by ljharb** [npm/feedback#349] [Website] [answered]
For example, http://npmjs.com/~es-shims says "25 packages" until you load more. By the time you get to the bottom, it will say the correct number, 76. The same thing happens on https://www.npmjs.com/~ljharb, which says "25", while the correct number is ~330 (or 297, if you don't count private packages)

Can these total counts please be made accurate? It's been broken for a long time, ever since one of the last redesigns.

**Comment by MylesBorins** [comment, marked as answer]
I've opened a bug with the registry team to track this. Will update if this work item is prioritized.

**Reply by MylesBorins** [reply]
Fyi this is now fixed

**Reply by MoOx** [reply]
Thanks!
